### DISCUSSION TITLE
Python/programmatic version of the bash/cli tutorial.

### DISCUSSION THREAD

**Discussion by ysig** [pinecone-io/canopy#235] [Q&A]
Hi,

is there a way to perform all the steps suggested by the tutorial, but all inside python?

Right now I'm trying to achieve the same goal using python but it would be great if I could do it by loading the library.

Thanks,

**Comment by gfleetwood** [comment]
This might help:

https://github.com/pinecone-io/canopy/blob/main/examples/canopy-lib-quickstart.ipynb
